demisto_sdk/utils/utils.py

```diff
@@ -104,3 +112,43 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return _retry
+
+
+def get_integration_params(project_id: str, secret_id: str) -> dict:
```

Comment:
lets make `project_id` default to `None`
and if it doesn't exist, take it from env var inside the func.

e..g;
```
if not project_id:
    project_id = os.getenv("env_var_name")
```

Reply:
@ilaner ?